EarlGrey/Action/GREYActions.m

```diff
@@ -262,6 +266,31 @@ @implementation GREYActions
 #pragma mark - Private
 
 /**
+ *  Set the UITextField text value directly, bypassing the iOS keyboard.
+ *  @param text     The text to be typed.
+ *
+ *  @return @c YES if the action succeeded, else @c NO. If an action returns @c NO, it does not
+ *          mean that the action was not performed at all but somewhere during the action execution
+ *          the error occured and so the UI may be in an unrecoverable state.
+ */
++ (id<GREYAction>)grey_actionForReplaceText:(NSString *)text {
+    return [GREYActionBlock actionWithName:[NSString stringWithFormat:@"Type \"%@\"", text]
```

Comment:
the action name should be a little different to differentiate it from typeText. How about something like:

[NSString stringWithFormat:"Replace with text: \"%@\"", text]